cg-spec/editors_draft.html

```diff
@@ -375,6 +375,21 @@ <h2>Overview</h2>
 -->
 </pre>
 
+    </section>
+    <section>
+      <h2>Common misconceptions</h2>
+
+      <p>People coming to RDF-star may have wrong ideas about what it is or how to use it. The purpose of this section is to dispel the most common misconceptions.
+
+      <p>RDF-star is <em>not</em> syntactic sugar for <a href="https://www.w3.org/TR/rdf11-mt/#reification">standard reification</a>. While the latter is a vocabulary fitting in the standard RDF model (abstract syntax), RDF-star extends that model (see <a href="#concepts"></a>) with a new construct, namely <a>quoted triples</a>. Note also that the two are not mutually exclusive: standard reification can still be used in conjunction with RDF-star.
+
+      <p>Unlike reified statements, RDF-star <a>quoted triples</a> are unique: wherever `&lt;&lt; :employee38 :jobTitle "Assistant Designer" &gt;&gt;` appears, it denotes one and the same thing. This impacts the modelling choices one has to make when using RDF-star; this is further discussed in <a href="#occurrences"></a>.
+
+      <p>Finally, <a>quoted triples</a> are referentially opaque. This means that two <a>quoted triples</a> containing distinct but equivalent terms (i.e. known to denote the same thing) are nonetheless considered to be different triples. <a href="#ref-opacity"></a> and the following sections discuss the rationale for this design, and how some form of referential transparency can still be added to RDF-star if and when needed.
```

Comment:
```suggestion
      <p>Finally, again like URIs, <a>quoted triples</a> are referentially opaque. This means that two <a>quoted triples</a> containing distinct but equivalent terms (i.e., coreferences, known to denote the same thing) are nonetheless considered to be different triples. <a href="#ref-opacity"></a> and the following sections discuss the rationale for this design, and how some form of referential transparency can still be added to RDF-star if and when needed.
```

Reply:
I think the "again like URIs/IRIs" is incorrect. IRIs in RDF (and in RDF-star when used outside quoted triples) are referentially transparent. They can be replaced by any synonymous term without changing the meaning of the graph.

Reply:
> [@pchampin] I think the "again like URIs/IRIs" is incorrect. IRIs in RDF (and in RDF-star when used outside quoted triples) are referentially transparent. They can be replaced by any synonymous term without changing the meaning of the graph.

Yeah, sorry, "lexically opaque" is what I was meaning, not "referentially opaque". My brain regularly flips these two. I imagine others' brains do the same, which is one reason I usually argue against using "referentially opaque" as a descriptor. Revising suggested change.